app/parser.go

```
@@ -0,0 +1,17 @@
+package app
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+)
+
+func ParseBody(r *http.Request, x interface{}) {
```

it should return an error in case it couldn't parse?